config.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+import os
+import pathlib
+import secrets
+
+import dotenv
+
+# TODO: These should be absolute (?) and configured by the user.
```

Comment:
disagree! suggest instead be opinionated and have code work well with in that opinion.  Document the choices made (eg 'photos go here' and 'listener is here') so folks have incite into the opinion

Reply:
Documented.